app-office/dia/Manifest

```diff
@@ -0,0 +1,9 @@
+AUX CVE-2019-19451.patch 291 BLAKE2B 99a0ad75b1a2be3e8aa023bc0268c9a5d30588efae020ec08e91d49faaf420a6b2cf0a0f4c57c5bf3c611b4331ac6a8b3ac7e1054ed7cb34fb95c55be660dd8c SHA512 c78b78bd4ce7bfab9babfc887fb98571849246d1ca78e6c69a4413c7881d30358e1287bd9b0d9ba50cbcc540cf5f0e28333b31f467dbe5579ad5c97148b3b512
+AUX dia-0.97.0-gnome-doc.patch 1961 BLAKE2B 6b6835a487e4b1ada0a1e67cb3ce123b0b4b3e67c86e9478ac0f401ae70f8f1381ef068a2ca6e116313db42d09bdb7abb523bf9a34856a1a39f7e4b0c9c3a84f SHA512 5200051f19d4a199f8c4e17e5f23c16ad4c13a9531b843be0a4d491aa8884efd4d90e6850c1cd1488e80b344ff18e5dc2b2ad7e57cb7417b6c574ad1e8d6f186
+AUX dia-0.97.2-underlinking.patch 1024 BLAKE2B 32fa473b806e740fb7c2b0e04c170c746bf523088e85dce281d0c87ef10c1542f1d77414dd49e6ce48b1e54c2e3e48d09c1820a8739bce6bb64539ba87b2485e SHA512 d45eb49c39227e9f11ce999083de0a83c60bfac2a5db2403e4bf083c275efdef6bd1adca678712253ec53f7b3d22a5c23f6f886839c32e6cb07a93c001f45e2e
+AUX dia-0.97.3-freetype_pkgconfig.patch 2574 BLAKE2B 870617b939e4ab7ecdfbf5bba4e78cee9f34ee10b1de2bb85380cc96d31c319132861b8ddd0e89a5a03cae2578588c9483ba62f0bf4006096c4074beef77cc4f SHA512 0e937bb17cb69d3f266f68658b11b209a913300d2366adfa4bb608212cd82f00d85272be784f0442d0dcba8659d020e5e848d2e02ee7310ec8cfd6c7170aa8b7
+AUX dia-musl-isinf.patch 327 BLAKE2B 507aa6a0dd452f07b7d64feaf7413b6aeeae38debda0442fba7dc69cde6829aa22603be54394e5cb7185d8c7c8c69288289bb9e5d28dab364e25f71809395a7e SHA512 6d3034a58e292ba4e748caa61fd314295b86f3b355cf2eb7783b752677cdb2cbaba59c9656441bfc68d31510d5de27608dd3252544b8c5833a3ed17acd8a1e81
+AUX dia-unregister-import.patch 967 BLAKE2B b19e745d5a6ec197ab5d8ee74200dd011f93c8a005a925816905b9d8cd239e473d24f948367d7d20bc682d042945a84e5627abff1f349528daf116a0aadda4b1 SHA512 24267507ff08ac589ec1015e55637586a9fd97802893a77849d1e7bce67135fbd3d55611e2c80be1d5f87b92406c95cd5fe484175037297ebe1dbc8eb509a7d4
```

Comment:
output of `pkgcheck scan --commits`:
```
app-office/dia
  PotentialStable: version 0.97.3: slot(0), stabled arches: [ amd64, hppa, ppc64, x86 ], potentials: [ ~alpha, ~arm, ~ia64, ~ppc ]
  UnnecessaryManifest: unnecessary files in Manifest: [ CVE-2019-19451.patch, dia-0.97.0-gnome-doc.patch, dia-0.97.2-underlinking.patch, dia-0.97.3-freetype_pkgconfig.patch, dia-0.97.3.ebuild, dia-musl-isinf.patch, dia-unregister-import.patch, metadata.xml ]
```
This overlay uses [`thin-manifests`](https://wiki.gentoo.org/wiki/Repository_format/metadata/layout.conf#thin-manifests)